engine/src/main/java/com/agiletec/aps/system/services/category/CategoryManager.java

```diff
@@ -42,63 +41,22 @@
 
 	private static final Logger _logger = LoggerFactory.getLogger(CategoryManager.class);
 
+	public static final String RELOAD_CATEGORY_REFERENCES_THREAD_NAME_PREFIX = "RELOAD_CATEGORY_REFERENCES_";
+

```

Comment:
Is this used anywhere else as a constant to add or remove from Cache? If so do you think it is worth creating a static CacheConstants file or a general Constants file with these?

If its not used elsewhere make private?

Reply:
It is used only in the CategoryAction (for the administration area) to check how many categories are changed compared to the total (to generate a progress bar that I do not see has been implemented in front-end).
It is not a constant managed by the cache. I do not know if it is appropriate to move it in a constant... what do you think? Maybe we can move it into the interface of the CategoryManager...

Reply:
The Interface seems like a better home to me than the implementation. It looks like the other interfaces have the constants so it would be consistent.